src/b.js

```diff
@@ -0,0 +1,2 @@
+const b = 1;;
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
'b' is assigned a value but never used  no-unused-vars